doc/API.md

```diff
@@ -4,13 +4,17 @@ Siad API
 The siad API is currently under construction. The deprecated way of doing
 things is documented at the end of the (incomplete) new documentation.
 
+All responses from the API contain a 'success' field (a bool) and an 'error'
+field (a string). If the query returns an error, the 'success' field will be
+set to false and there will be an error message. If no error occurs, 'success'
+will be set to true and the error field will be empty.
```

Comment:
This is currently not implemented. It was more of a proposal. I was wondering if we could wrap it into the writeError and writeJSON functions